pygfx/renderers/wgpu/pointsrender.py

```diff
@@ -2,8 +2,49 @@
 
 from . import register_wgpu_render_function
 from ._shadercomposer import Binding, WorldObjectShader
+from ._utils import to_vertex_format, to_texture_format
 from ...objects import Points
 from ...materials import PointsMaterial, GaussianPointsMaterial
+from ...resources import Texture, TextureView
+
+
+def handle_colormap(geometry, material, shader):
```

Comment:
This `handle_colormap` function (in pointsrender.py`) is also used in the mesh and line renderers.